examples/vrp-litmus/Dockerfile-vrp

```diff
@@ -1 +1 @@
-FROM envoyproxy/envoy-google-vrp-dev:latest
+FROM envoyproxy/envoy-google-vrp-dev:latest@sha256:fcf8e391bafa0169b3f94773f7d1306169752ae133f75d6ce9f66ad5336035f5
```

Comment:
apologies - i should have been more explicit - this also gets tested in ci - i have an attempt somewhere to shift it to an example - but for now this also gets loaded and tested

thinking on the windows one - it doesnt get tested in ci - but the point about the dev images is that they are intended to pull in the latest version - so not really pinnable